src/js/tech/middleware.js

```diff
@@ -244,7 +244,9 @@ function executeRight(mws, method, value, terminated) {
  *         A {@link Player} instance.
  */
 export function clearCacheForPlayer(player) {
-  middlewareInstances[player.id()] = null;
+  if (middlewareInstances.hasOwnProperty(player.id())) {
+    middlewareInstances[player.id()] = null;
```

Comment:
I wonder if `delete` is preferable, if the object having null properties was the issue being addressed

Reply:
Let's go with `delete` here

Reply:
ok, commit is pushed

Reply:
not sure if - at this point - code should just delete without checking whether `middlewareInstances` has that property or not.